core/controllers/admin.py

```diff
@@ -1226,6 +1270,115 @@ def _generate_dummy_classroom(self) -> None:
+                # of interaction_id.
+                assert state.interaction.id is not None
+                solution = state_domain.Solution.from_dict(
+                    state.interaction.id, solution_dict)
+                state.update_interaction_solution(solution)
+                state.update_interaction_hints(hints_list)
+                state.update_interaction_customization_args({
+                    'placeholder': {
+                        'value': {
+                            'content_id': content_id_generator.generate(
+                                translation_domain.ContentType.CUSTOMIZATION_ARG, # pylint: disable=line-too-long
+                                extra_prefix='placeholder'),
+                            'unicode_str': 'Enter text here',
+                        },
+                    },
+                    'rows': {'value': 1},
+                    'catchMisspellings': {'value': False}
+                })
+                # Here, state is a State domain object and it is created using
+                # 'create_default_state' method. So, 'state' is a default_state
+                # and it is always going to contain a default_outcome. Thus to
+                # narrow down the type from Optional[Outcome] to Outcome for
+                # default_outcome, we used assert here.
+                assert state.interaction.default_outcome is not None
+                state.interaction.default_outcome.labelled_as_correct = True
+                state.interaction.default_outcome.dest = None
+                suggestion_change: Dict[
+                    str, Union[str, float, question_domain.QuestionDict]
+                ] = {
+                    'cmd': (
+                        question_domain
+                        .CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION),
+                    'question_dict': {
+                        'id': '',
+                        'version': 0,
+                        'question_state_data': state.to_dict(),
+                        'language_code': 'en',
+                        'question_state_data_schema_version': 1,
+                        'linked_skill_ids': [skill_id],
+                        'inapplicable_skill_misconception_ids': [],
+                        'next_content_id_index': (
+                            content_id_generator.next_content_id_index)
+                    },
+                    'skill_id': skill_id,
+                    'skill_difficulty': 0.3
+                }
+
+                suggestion = suggestion_services.create_suggestion(
+                feconf.SUGGESTION_TYPE_ADD_QUESTION,
+                feconf.ENTITY_TYPE_SKILL,
+                skill_id, 1,
+                self.user_id, suggestion_change, 'test description')
+
+                (
+                    suggestion_services
+                    .update_question_contribution_stats_at_submission(
+                        suggestion))
```

Comment:
Why do you add parenthesis around this expression?

Reply:
I have added parenthesis because this was exceeding max line character limit, so to distribute it into 2 lines.